Src/GoogleApis.Auth.DotNet4.Tests/OAuth2/DefaultCredentialProviderTests.cs

```diff
@@ -192,6 +192,9 @@ public void SetUp()
             Assert.IsInstanceOf(typeof(ServiceAccountCredential), scopedCredential.UnderlyingCredential);
             Assert.IsFalse(scopedCredential.IsCreateScopedRequired);
             CollectionAssert.AreEqual(scopes, ((ServiceAccountCredential) scopedCredential.UnderlyingCredential).Scopes);
+
+            var scopedCredential2 = credential.CreateScoped("scope1", "scope2");
```

Comment:
1. Do you need to declare a new variable here, or is it just better to use scopedCredential? I prefer the latter.
2. Is it the right place to put this test (under a test that its name is ...FromEnvironmentVariable)?